packages/mobile/src/navigator/NavigationService.ts

```diff
@@ -88,6 +88,31 @@ export function navigate<RouteName extends keyof StackParamList>(
     })
 }
 
+export function navigateIf<RouteName extends keyof StackParamList>(
```

Comment:
Do I understand correct, that the reason to have this method is that we do not want to call `ensureNavigator` somewhere else in the code if we need to check current navigation state?

Reply:
Yes, it was. It was related to the `navigateToExchangeScreenIfReviewing` above, but if we remove that we can remove this too.